src/Multilingual-Encodings.package/Unicode.class/class/isConnectorPunctuation..st

```diff
@@ -0,0 +1,7 @@
+character classification
+isConnectorPunctuation: char
+	| index |
+	index := char charCode + 1.
```

Comment:
There is a recurring pattern to these that I think should be factored out.
e.g.  ^self isCharacter: aCharacter category: Pc

Reply:
I agree — except that I would call it isCharacter: ch inCategory: cat .